Alignment/OfflineValidation/src/TrackerValidationVariables.cc

```diff
@@ -127,6 +127,7 @@ void TrackerValidationVariables::fillHitQuantities(reco::Track const & track, st
                     gVDirection = surface.toGlobal(lVDirection);
 
         if (IntSubDetID == PixelSubdetector::PixelBarrel ||
+          IntSubDetID == PixelSubdetector::PixelEndcap ||
```

Comment:
@VinInn 
shouldn't the Forward Pixel topology require a different treatment of the local x and y orientations as in the other pre-existing method using using the trajectory: https://github.com/VinInn/cmssw/blob/1258c841813a3b05fa13c505931c119f1008f8d4/Alignment/OfflineValidation/src/TrackerValidationVariables.cc#L303-L323 ?